src/App.style.js

```diff
@@ -0,0 +1,27 @@
+import { NavLink } from 'react-router-dom';
+import styled, { injectGlobal } from 'styled-components';
+
+export const NavBar = styled.div`
+background-color: #333;
+overflow: hidden;
+`;
+
+export const HeadLink = styled(NavLink)`
+float: left;
+color: #f2f2f2;
+text-align: center;
+padding: 14px 16px;
+text-decoration: none;
+font-size: 17px;
+`;
+
+injectGlobal`
```

Comment:
create another file that's called `global.style.js` and then import it in `App.js`

Reply:
Thanks for the feedback.

I'll do it, right now!